bp_common/src/include/bp_common_me_if.vh

```diff
@@ -406,65 +453,16 @@ typedef enum logic
  * addr is the memory address from the cache miss
  * payload is data sent to mem and returned to cce unmodified
  */
-`define declare_bp_cce_mem_cmd_s(addr_width_mp) \
-  typedef struct packed                                    \
-  {                                                        \
-    bp_lce_cce_req_type_e                        msg_type; \
-    logic [addr_width_mp-1:0]                    addr;     \
-    bp_cce_mem_cmd_payload_s                     payload;  \
-    bp_lce_cce_req_non_cacheable_e               non_cacheable; \
-    bp_lce_cce_nc_req_size_e                     nc_size;       \
+`define declare_bp_cce_mem_cmd_s(addr_width_mp, data_width_mp)  \
+  typedef struct packed                                         \
+  {                                                             \
+    logic [data_width_mp-1:0]                    data;          \
+    bp_cce_mem_cmd_payload_s                     payload;       \
+    bp_lce_cce_uc_req_size_e                     uc_size;       \
```

Comment:
Two thoughts:
1) Let's not use any lce_cce fields in these structs
2) Possibly uc_size is the wrong concept? For example: If we end up doing hierarchical coherence, we may need to send a req size to the next level cache depending on the block size.  So just req_size?